src/validationplots/plots.jl

```diff
@@ -144,3 +144,54 @@ function diagnosticplots(fm::MaximumLikelihoodEVA)::Gadfly.Compose.Context
 
     return gridstack([f1 f2; f3 f4])
 end
+
+
+"""
+    mrl(y::Vector{<:Real}, steps::Int = 100)::DataFrame
+
+Compute the mean residual life from vector `y` using the set of thresholds from the `minimum(y)` value to the second-to-last larger value in `steps` steps.
+"""
+function mrl(y::Vector{<:Real}, steps::Int = 100)::DataFrame
```

Comment:
The other plot data functions are named this way :
`probabilityplot_data()` is associated with `probabilityplot()`
Therefore, this function should be named this for consistency:
`mrlplot_data()`
We could also rename all the other plots if you prefer this convention.